server/cmd/mmctl/commands/team.go

```diff
@@ -180,6 +180,7 @@ func deleteTeam(c client.Client, team *model.Team) (*model.Response, error) {
 }
 
 func archiveTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	var errorString multierror.Error
```

Comment:
Let's stick to the naming schema for other methods.
```suggestion
	var result *multierror.Error
```